assets/scripts/scatter/density_hexbin_with_hist.py

```diff
@@ -0,0 +1,28 @@
+# %%
+from __future__ import annotations
+
+import matplotlib.pyplot as plt
+import plotly.express as px
+import plotly.io as pio
+
+import pymatviz as pmv
+from pymatviz.data.regression import y_pred, y_true
+
+
+px.defaults.template = "pymatviz_white"
+pio.templates.default = "pymatviz_white"
+
+
+# %% Configure matplotlib and load test data
+plt.rc("font", size=14)
+plt.rc("savefig", bbox="tight", dpi=200)
+plt.rc("axes", titlesize=16, titleweight="bold")
+plt.rc("figure", dpi=200, titlesize=20, titleweight="bold")
+plt.rcParams["figure.constrained_layout.use"] = True
+
```

Comment:
would be good to centralize these settings instead of repeating them in 15 asset scripts. if you import a file which has these commands at the top level, the settings will take effect

Reply:
Yes of course, sorry I'm currently playing with Network UPS Tools trying to set it up for my toy server setup, I would get my hands on this later.

Thanks for the help, it's super helpful :)